MongoDB takes too long to delete old revoked uvci

anversoft opened the pull request:
When the revoked certificates are too many (> 50000) MongoDB takes too long to delete the certificates (+ 2h), I found it more efficient to completely delete the database and download the new certificates instead of deleting them one by one, the clean up threshold can be set using the env variable `VC19_MONGODB_CLEAN_THRESHOLD` (default 50000).

astagi commented:
Ciao @anversoft,

thank you for this PR, great catch! I think that a better and most efficient solution could be using `deleteMany` to delete entries, something like

```
this._dbModel.deleteMany({ id: { $in: deletedRevokedUvci }})
```

Try and let me know, in case could you please rewrite that part using `deleteMany`?

anversoft commented:
Ok, i tried a few times and now it works fine. I removed the env variable `VC19_MONGODB_CLEAN_THRESHOLD `because with `deleteMany `it is no longer necessary but I added the env variable `VC19_UPDATE_HOURS `to force the check of the clr since the revoked certificates are updated every 5/6 hours

astagi commented:
Great @anversoft ! Thank you so much!
